base/doc/ltnews41.tex

```diff
@@ -452,6 +452,25 @@ \subsection{\pkg{array}:\ Improve preamble setup code for \texttt{p} and friends
 \githubissue{1585}
 
 
+\subsection[\pkg{varioref}:\ How to make \cs{reftex...}\ empty]
+           {\pkg{varioref}:\ How to make \cs{reftexafter} and friends empty}
```

Comment:
Needs one more   \   after the  }

Reply:
don't think so. Why? the `\ `is only needed after punctuations as they leak in the TUB class and produce extra space.

Reply:
One of these uses   }\ 
the other uses   no  \ 

I am really querying why they are different as neither involves any punctuation.

Reply:
well, "." is a punctuation, isn't it? even if it is not meant to be a punctuation it seems to leak.

Reply:
Yes, I guess it is, but not so clear when looking at the .tex!